install/kubernetes/tetragon/templates/_container_tetragon.tpl

```diff
@@ -70,14 +70,9 @@
 {{- else if .Values.tetragon.grpc.enabled }}
```

Comment:
I think it should check `{{- .Values.tetragon.healthGrpc.enabled }}`, right?

Reply:
Thanks, fixed!